docs/troubleshooting-guide/sandboxing.md

```diff
@@ -0,0 +1,76 @@
+# Sandboxing
+
+[Sandboxing data][sandboxing-your-data] gives some people access to only a subset of the data. (The term comes from the practice of putting children in a sandbox to play safely.) To implement sandboxing, Metabase runs a query that filters rows and/or selects a subset of columns from a table; the user's query then runs on the initial query's result.
+
+Several databases did not support [common table expressions][cte] when sandboxing was added to Metabase, so we implemented it using subqueries. Suppose you use the Notebook Editor to create a query like:
+
+```
+select * from orders
+where price > 100.00
+```
+
+If the `orders` table is sandboxed by the user's ID, what we actually run is something like this:
+
+```
+SELECT
+  made_up_name_01.customer_id AS customer_id,
+  made_up_name_01.price AS price,
+  made_up_name_01.whatever AS whatever
+FROM
+  (
+    SELECT
+      orders.customer_id AS customer_id,
+      orders.price AS price,
+      orders.whatever AS whatever
+    FROM
+      orders 
+    WHERE
+      orders.customer_id = {{user_id}}
+  )
+  made_up_name_01
+WHERE
+  made_up_name_01.price > 100
+```
+
+Metabase creates a unique temporary name like `made_up_name_01` to make sure the query doesn't actually pull in data from an existing table. The curly braces `{{...}}` around `user_id` show that it is bound to an attribute from the user properties.
+
```

Comment:
To provide more context, we should include links to data permissions and sandboxing articles in Learn.